Jenkinsfile

```
@@ -16,84 +16,82 @@ pipeline {
                 sh 'docker build --target test -f Dockerfile.swarm .'
             }
         }
-        stage('Build') {
-            steps {
-                sh '''docker build \\
-                    --target build \\
-                    --build-arg environment=production \\
-                    --build-arg react_app_router=browser \\
-                    --build-arg public_url="/" \\
-                    -f Dockerfile.swarm \\
-                    .'''
-            }
-        }
-        stage('Release') {
-            when { 
-                branch 'master'
-            }
-            steps {
-                sh 'sh generate-release-w-docker.sh'
-            }
-        }
-        stage('Build final version images') {
-            when {
-                expression {
-                    return isVersionTag(readCurrentTag())
-                }
-            }
-            steps {
-                // TODO: remove build differences based on environments to allow reducing the
-                // number of different images. Maybe `demo` and `production` cold be enought
-                sh '''docker build \
-                    -t "local-doppler-webapp:production-commit-${GIT_COMMIT}" \\
-                    --build-arg environment=production \\
-                    --build-arg react_app_router=browser \\
-                    --build-arg public_url="/" \\
-                    --build-arg version=production-${TAG_NAME}+${GIT_COMMIT} \\
-                    -f Dockerfile.swarm \\
-                    .'''
-                sh '''docker build \
-                    -t "local-doppler-webapp:demo-commit-${GIT_COMMIT}" \\
-                    --build-arg environment=demo \\
-                    --build-arg react_app_router=browser \\
-                    --build-arg public_url="/" \\
-                    --build-arg version=demo-${TAG_NAME}+${GIT_COMMIT} \\
-                    -f Dockerfile.swarm \\
-                    .'''
-                 sh '''docker build \
-                    -t "local-doppler-webapp:int-commit-${GIT_COMMIT}" \\
-                    --build-arg environment=int \\
-                    --build-arg react_app_router=browser \\
-                    --build-arg public_url="/" \\
-                    --build-arg version=int-${TAG_NAME}+${GIT_COMMIT} \\
-                    -f Dockerfile.swarm \\
-                    .'''
-                    sh '''docker build \
-                    -t "local-doppler-webapp:qa-commit-${GIT_COMMIT}" \\
-                    --build-arg environment=qa \\
-                    --build-arg react_app_router=browser \\
-                    --build-arg public_url="/" \\
-                    --build-arg version=qa-${TAG_NAME}+${GIT_COMMIT} \\
-                    -f Dockerfile.swarm \\
-                    .'''
-            }
-        }
         stage('Publish final version images in dopplerdock') {
```

Dentro de esta sección se hacen los publish de INT, master y V1, creo que _final version_  no es un nombre adecuado, que opinás de:

```suggestion
        stage('Publish images to Docker Hub') {
```